src/schema/v2/me/myCollectionInfo.ts

```diff
@@ -181,14 +181,19 @@ export const myCollectionInfoFields = {
         args
       )
 
+      // TODO: Remove this once all clients pass includePersonalArtists correctly
+      // This is a hack and currently we are defaulting to true if the query comes from the artwork form (passing a size of 100)
+      const includePersonalArtists =
+        args.size === 100 ? true : args.includePersonalArtists
```

Comment:
Open to suggestions to avoid this hack.

Reply:
I think allowing old clients to stay `false` would be the proper way to go. Since this can easily be fixed on Force, the issue would only be on Eigen and that can easily be remedied by having a version that has been tested thoroughly for this behaviour instead of trying to make it compatible with those versions. What do you think? Because users on these versions are already using the app without issues regarding this. Instead let us ensure the next release sends this arg.

Reply:
The issue is that with the current change users of the app would only see artists that are not personal (non-public) and currently we need to show all artists for one query (insights) and only non-personal (public) artists for another query (artwork form). 

If we don't use a hack like this, we would need to wait with this change until all users have updated their apps to the new version which will be released next year.

Or is there another way I don't see? 🤔

Reply:
I cannot think of any other way. Since this is documented and there is a followup ticket to update this, I think we can live with this for now.